client/state/selectors/get-post-share-scheduled-actions.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { enrichPublicizeActionsWithConnections } from 'state/selectors/utils/';
+
+/**
+ * Return a share-scheduled-actions array propagaring data from publicize connections.
+ *
+ * @param {Object} state Global state tree
+ * @param {Number} siteId Site ID
+ * @param {Number} postId Post ID
+ * @return {Array} share publihed actions array
+ */
+export default function getPostShareScheduledActions( state, siteId, postId ) {
```

Comment:
Maybe consider memoizing this using [create-selector](https://github.com/Automattic/wp-calypso/tree/master/client/lib/create-selector)